projects/www/src/app/components/docs/code-tabs.component.ts

```diff
@@ -0,0 +1,64 @@
+import {
+  Component,
+  OnInit,
+  ElementRef,
+  inject,
+  signal,
+  viewChild,
+} from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MatTabsModule } from '@angular/material/tabs';
+import { DomSanitizer, SafeHtml } from '@angular/platform-browser';
+import { CodeExampleComponent } from './code-example.component';
+
+@Component({
+  selector: 'ngrx-code-tabs',
+  standalone: true,
+  imports: [CommonModule, MatTabsModule, CodeExampleComponent],
+  template: `
+    <div #content style="display: none"><ng-content></ng-content></div>
+
+    <mat-tab-group [preserveContent]="true">
+      @for (tab of tabs(); track tab) {
+      <mat-tab [label]="tab.header">
+        <ngrx-code-example [innerHTML]="tab.code"> </ngrx-code-example>
```

Comment:
Maybe not the best implementation, but it works.
The idea is to reuse the `ngrx-code-example` in the content, and re-render it here using it's own rendered HTML.
The `ngrx-code-example` is re-used here to have the copy functionality.